src/main/sd/init.c

```diff
@@ -0,0 +1,310 @@
+#include "sd.h"
+#include "debug.h"
+#include "pin.h"
+#include "spi.h"
+
+static bool sd_card_initialized = false;
+uint8_t csd_register[16];
+static uint8_t sd_state;
+mext2_command* command;
+
+#define SD_NOT_DETERMINED   0
+#define SD_V1X              1
+#define SD_V2X              2
+#define SD_V2XHCXC          3
+
+uint8_t sd_version = SD_NOT_DETERMINED;
+
+
+#define COMMAND_SOFTWARE_RESET      (uint8_t)0x00    //CMD0
```

Comment:
Hmmm... Wouldn't it be better to create separate file in which all sd commands will be defined instead of defining them in both rw.h and init.h. But that's open to discussion since it could add some overhead and all

Reply:
init and rw dont have any common comands, but ok, I have created separate file with all commands